src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/visitbalances/VisitBalanceNomisApiService.kt

```diff
@@ -8,6 +8,7 @@ import org.springframework.web.reactive.function.client.awaitBody
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.helpers.awaitBodyOrNullForNotFound
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomisprisoner.model.CreateVisitBalanceAdjustmentRequest
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomisprisoner.model.CreateVisitBalanceAdjustmentResponse
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomisprisoner.model.UpdateVisitBalanceRequest
```

Comment:
I don't get why there are additional imports on this pr 🤷

Reply:
It is now picking up the request object from nomisprisonapi - previously I had hardcoded it until I had completed the change